Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs

```diff
@@ -189,8 +190,8 @@ public void ApplyConfiguredPerformanceSettings()
         {
 #if OCULUSINTEGRATION_PRESENT
             UnityEngine.XR.XRSettings.eyeTextureResolutionScale = resolutionScale;
-            OVRManager.cpuLevel = CPULevel;
-            OVRManager.gpuLevel = GPULevel;
+            OVRManager.suggestedCpuPerfLevel = CPULevel;
```

Comment:
Is this a change specific to a certain Oculus package? Or is it backwards compatible?